modules/blazingsql/cluster.js

```diff
@@ -0,0 +1,75 @@
+const cluster = require('cluster');
+const { BlazingContext, UcpContext } = require('@rapidsai/blazingsql');
+const { Series, DataFrame } = require('@rapidsai/cudf');
+
+const createTable = 'createTable';
+const runQuery = 'runQuery';
+const queryRan = 'queryRan';
+
+const numberOfWorkers = 2;
+
+const ucp_context = new UcpContext();
+
+if (cluster.isMaster) {
+  const workers = Array(numberOfWorkers);
+  for (let i = 0; i < numberOfWorkers; ++i) {
+    workers[i] = cluster.fork();
+  }
+
+  const bc = new BlazingContext(Object.keys(cluster.workers).map((id) => {
+    return {
+      workerId: id,
+      ip: 'localhost',
+      port: 8000,
+      ucpContext: ucp_context,
+    };
+  }));
+
+  const a = Series.new([1, 2, 3]);
+  const b = Series.new([1, 2, 3]);
+  const df = new DataFrame({ 'a': a, 'b': b });
+
+  workers.forEach((w) => {
+    w.send({ operation: createTable, tableName: 'test_table', dataframe: df });
```

Comment:
I think we'll need to send the table in binary form here, then we can deserialize it on the child side:

```suggestion
    w.send({
    operation: createTable,
    tableName: 'test_table',
    dataframe: df.toArrow().serialize()
    });
```

Reply:
Thanks, yeah, was going to poke you abot this on monday.